wireapp/wire-android
# feat: Add pagination in searched messages result screen (WPB-5498)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-5498" title="WPB-5498" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />WPB-5498</a>  Add pagination results
  </td></table>
  <br />
 






















----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

There was a bit of a performance issue when search result was extensive.

### Solutions

Added pagination to search result (now its so blazing fast that sometimes the loading icon doesn't even show 😢 )

### Dependencies (Optional)

Needs releases with:

- [X] [Kalium - feat: Add pagination in searched messages result screen (WPB-5498)](https://github.com/wireapp/kalium/pull/2277)

### Testing

#### Test Coverage (Optional)


#### How to Test

- Open App
- Open Conversation
- Open Conversation Details
- Open Search
- Type a search query with many results
- Verify that results are now faster and paginated
